<launch>
  <!--remap from="odom" to="odometry/filtered"/-->
  <!--remap from="cmd_vel" to="nav_vel"/-->
  
  <node name="move_base"
        pkg="move_base"
        type="move_base"
        clear_params="true"
	output="screen">

    <rosparam command="load"
              file="$(find metu_navigation)/config/move_base/move_base.yaml"/>

    <!-- costmaps -->
    <rosparam command="load"
              ns="global_costmap"
              file="$(find metu_navigation)/config/move_base/costmaps/costmap_common.yaml"/>
    <rosparam command="load"
              ns="local_costmap"
              file="$(find metu_navigation)/config/move_base/costmaps/costmap_common.yaml"/>
    <rosparam command="load"
              file="$(find metu_navigation)/config/move_base/costmaps/local_costmap.yaml"/>
    <rosparam command="load"
              file="$(find metu_navigation)/config/move_base/costmaps/global_costmap.yaml"/>

    <!-- planners -->
    <rosparam command="load"
              file="$(find metu_navigation)/config/move_base/planners/local_planner.yaml"/>
    <rosparam command="load"
              file="$(find metu_navigation)/config/move_base/planners/global_planner.yaml"/>
 
  </node>
<node
    name="tf_footprint_base"
    pkg="tf"
    type="static_transform_publisher"
    args="0 0 0 0 0 0 base_link base_footprint 40" />
  
</launch>
